docs/ble.md

```diff
@@ -25,21 +38,22 @@ treat them like decimals.*
 Web IDL
 -------
 This IDL provides an overview of the interface; see below for documentation of
-specific API functions.
+specific API functions.  We also have a short document explaining [ZJS WebIDL conventions](Notes_on_WebIDL.md).
+
 
 ```javascript
 // require returns a BLE object
 // var ble = require('ble');
 
-[NoInterfaceObject]
+[ReturnFromRequire]
 interface BLE: EventEmitter {
     void disconnect(string address);
     void startAdvertising(string name, string[] uuids, string url);
```

Comment:
We also don't have strings in Web IDL, we have DOMString, ByteString and USVString. Thus here you probably want

`startAdvertising(USVString name, sequence<DOMString>, USVString url)`